src/orchard/misc.clj

```diff
@@ -188,3 +189,9 @@
   [x]
   (when (and (coll? x) (not (lazy-seq? x)))
     (count x)))
+
+(defn safe-read-edn
+  "Read the string `s` in EDN format in s safe way."
```

Comment:
s safe -> a safe?

You might want to explain what "safe" means in this context and add `:added` metadata to the function.

Reply:
Fixed. Regarding the meta data. Do you want this on all functions? I added :added metadata to the new namespaces. But should all functions inside this namespace have this metadata as well? Also for private functions? I have nothing against that, I actually like seeing this in *cider-doc*. I'm just wondering where to stop.

Reply:
Well, I just added it everywhere so we can use it in cider-doc. It's not as verbose as I initially thought.